mainWidget/main_widget.yaml

```diff
@@ -582,7 +600,7 @@ slots:
                       - component: oh-repeater
                         config:
                           fetchMetadata: semantics,metadata,widgetOrder
-                          filter: loop.floorItem.name==vars.objVar.floor
+                          filter: (loop.floorItem.metadata) && (loop.floorItem.name == vars.objVar.floor) && (loop.floorItem.metadata.semantics.value).includes("Floor")
```

Comment:
Why should we need that change ?
(loop.floorItem.metadata.semantics.value).includes("Floor") was already checked setting objVar.floor
Seems to be an redundant check to me.

Reply:
I am fine with checking if metadata is existing, but please remove redundant semantic value check.

Reply:
If the objVar is not set, all items are rendered into containers. You don't see it because it is normally invisible as long as no objVar is set.
If I make the container permanently visible:

Actual in Repo:

>                           filter: loop.floorItem.name==vars.objVar.floor

![grafik](https://user-images.githubusercontent.com/79790877/218313417-badff1d1-12ce-4a88-90ae-49e22d76f129.png)
and so on...........

My new filter. There was still an error in the first commit. It is adjusted here. If desired I commit the change.

>                           filter: ((loop.floorItem.metadata) && (loop.floorItem.metadata.semantics.value).includes("Floor")) && (loop.floorItem.name == vars.objVar.floor)

![grafik](https://user-images.githubusercontent.com/79790877/218313272-9077d726-5885-48bc-b75a-a30145461777.png)

Only the possible floors are rendered. Until objVar.floors is set.

Reply:
Ok, please commit you new filter expression

Reply:
fixes with [1242914](https://github.com/hmerk/main_widget/pull/37/commits/12429143d7b50a55d9280fc9bf36ddaef431d2ab)